docs/source/io_formats/track.rst

```diff
@@ -4,18 +4,33 @@
 Track File Format
 =================
 
-The current revision of the particle track file format is 2.0.
+The current revision of the particle track file format is 3.0.
 
 **/**
 
 :Attributes: - **filetype** (*char[]*) -- String indicating the type of file.
              - **version** (*int[2]*) -- Major and minor version of the track
                file format.
-             - **n_particles** (*int*) -- Number of particles for which tracks
-               are recorded.
-             - **n_coords** (*int[]*) -- Number of coordinates for each
-               particle.
 
 :Datasets:
-           - **coordinates_<i>** (*double[][3]*) -- (x,y,z) coordinates for the
-             *i*-th particle.
+           - **track_<b>_<g>_<p>** (Compound type) -- Particle track information
+             for source particle in batch *b*, generation *g*, and particle
+             number *p*. particle. The compound type has fields ``r``, ``u``,
+             ``E``, ``time``, ``wgt``, ``cell_id``, and ``material_id``, which
```

Comment:
Sorry for a late-in-the-game comment -- Is it worth adding cell instance to the set of data on the tracks?

Reply:
Yeah, it probably is. That will increase the size of the underlying struct from 80 bytes to 84 bytes, or a 5% increase. I'd say it's worthwhile to have that in so I'll go ahead and add it.